.changeset/cold-squids-explain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": patch
+---
+
+Fix on @cypher directive fields in connections
```

Comment:
Always make sure to escape directives otherwise we end up tagging users in our changelogs! 🙃 

```suggestion
Fix on `@cypher` directive fields in connections
```